modules/swagger-codegen/src/main/resources/typescript-angular/README.mustache

```diff
@@ -2,42 +2,41 @@
 
 ### Building
 
-To build an compile the typescript sources to javascript use:
+To install the required dependencies and to build the typescript sources run:
 ```
 npm install
-npm run build
 ```
 
 ### publishing
 
-First build the package than run ```npm publish```
+First build the package than run ```npm publish dist``` (don't forget to specify the `dist` folder!)
 
 ### consuming
 
-navigate to the folder of your consuming project and run one of next commando's.
+navigate to the folder of your consuming project and run one of next commands.
 
 _published:_
 
 ```
 npm install {{npmName}}@{{npmVersion}} --save
 ```
 
-_unPublished (not recommended):_
+_without publishing (not recommended):_
 
 ```
-npm install PATH_TO_GENERATED_PACKAGE --save
+npm install PATH_TO_GENERATED_PACKAGE/dist --save
 ```
 
 _using `npm link`:_
```

Comment:
add a note about an issue in angular CLI that prevents using linked libraries on Windows, see https://github.com/angular/angular-cli/issues/8284

Reply:
✅